src/routes/api/+server.js

```diff
@@ -1,8 +1,18 @@
-import { json } from '@sveltejs/kit';
+import prisma from '$lib/prisma';
 
 /** @type {import('./$types').RequestHandler} */
 export async function POST({ request }) {
-    const { json } = await request.json();
-    console.log(json)
+    const data = await request.json();
+    await prisma.app.upsert({
+        where: {
+            appId: data.appId
+        },
+        create: data,
```

Comment:
This will not work.  You need to add code for handling the app and the listings separately. See the last example in this ChatGPT question: https://chat.openai.com/share/e9088187-f951-481c-8f4a-1d22241d7cb5